### DISCUSSION TITLE
Digest auth support

### DISCUSSION THREAD

**Discussion by marek-benes** [nodejs/undici#1611] [Ideas]
Hello, do you plan to add digest auth support?

Thank you.

**Comment by mcollina** [comment]
I think this could be easily added on top. However it might be possible that this is needed for `fetch()`, in that case it'll happen.

**Comment by aristos32** [comment]
Hello, do you plan to add digest auth support?

**Reply by mcollina** [reply]
I think we might have to for fetch(). wdyt @KhafraDev?

**Reply by KhafraDev** [reply]
fetch doesn't implement it. A package like [digest-fetch](https://www.npmjs.com/package/digest-fetch?activeTab=readme) should work instead.

**Reply by 0-don** [reply]
would be nice to get support for this
